source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -166,23 +168,34 @@ Install |CL| with Advanced Installation
 #. Complete the remaining steps of :guilabel:`Required Options` to 
    to install |CL|. Complete any :guilabel:`Advanced Options` as desired.
     
-Ways to boot |CL|
-*****************
+Three Methods to Boot |CL|
+*************(************
+
+Although we installed |CL| last, Ubuntu is still the default boot OS.  
+There are three methods to boot |CL|:
 
-Although we installed |CL| last, Ubuntu is still the default boot OS.  There are several ways to boot |CL|:
+#. Make systemd-boot, the boot loader that |CL| uses, the default
```

Comment:
Please revise this sentence. Something along the lines of:

Make `systemd-boot` the default bootloader for |CL|, and chain-boot GRUB for another distro.